<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>music</class>
 <widget class="QMainWindow" name="music">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="musicList">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>281</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>511</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="prevBtn">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="playBtn">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextBtn">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="modeBtn">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="listBtn">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>561</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="currentTime">
       <property name="text">
        <string>0:0:0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="progressBar">
       <property name="styleSheet">
        <string notr="true">
QSlider::groove:horizontal{ 
		height: 12px; 
		left: 0px; 
		right: 0px; 
		border:0px;    
	    border-radius:6px;    
		background:rgba(0,0,0,50);
 } 

 QSlider::handle:horizontal{ 
		width:  50px; 
		height: 50px; 
		margin-top: -20px; 
		margin-left: 0px; 
		margin-bottom: -20px; 
		margin-right: 0px; 
 		border-image:url(:/res/images/setting_slider_handle.png);
} 

QSlider::sub-page:horizontal{
	   background:rgba(80,166,234,1);
}

</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="totalTime">
       <property name="text">
        <string>0:0:0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>853</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
